etcdctl/ctlv3/command/ep_command.go

```diff
@@ -102,12 +115,16 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
 			if err == nil || err == rpctypes.ErrPermissionDenied {
 				fmt.Printf("%s is healthy: successfully committed proposal: took = %v\n", ep, time.Since(st))
 			} else {
-				fmt.Printf("%s is unhealthy: failed to commit proposal: %v\n", ep, err)
+				errc <- fmt.Errorf("%s is unhealthy: failed to commit proposal: %v", ep, err)
 			}
-		}(cfg)
+		}(cfg, errc)
 	}
 
 	wg.Wait()
+	close(errc)
+
+	// exit with non-zero exit status in case of errors.
+	os.Exit(<-errCountc)
```

Comment:
the exit code shouldn't change based on the number of errors; `ExitWithError(ExitError, fmt.Errorf("unhealthy cluster"))` would be more consistent with the rest of etcdctl